src/components/human-impact-layers/human-impact-layers-component.jsx

```diff
@@ -10,10 +10,9 @@ import { VIEW_MODE } from  'constants/google-analytics-constants';
 
 const HumanImpactLayers = ({ map, rasters, setRasters, setLayerVisibility, activeLayers, addLayerAnalyticsEvent, removeLayerAnalyticsEvent }) => {
   const humanImpactLayerActive = activeLayers.find(l => l.title === LAND_HUMAN_PRESSURES_IMAGE_LAYER);
-  const alreadyChecked = humanImpactLayerActive && (humanPressuresLandUse.reduce((acc, option) => ({
+  const alreadyChecked = humanImpactLayerActive && ((humanPressuresLandUse.reduce((acc, option) => ({
     ...acc, [option.value]: rasters[option.value]
-  // eslint-disable-next-line no-mixed-operators
-  }), {})) || {};
+  }), {})) || {});
```

Comment:
I think I disabled this line for a reason 🤔 there's a bug occurring now on the sidebar, take a look, it might be related to this line ^ 

![small-screencast](https://user-images.githubusercontent.com/6136899/61622779-5e004700-ac6d-11e9-8c93-d2a25d9e87e6.gif)

Reply:
indeed, I can't reproduce the bug when I brought back the changes:
```
const alreadyChecked = humanImpactLayerActive && (humanPressuresLandUse.reduce((acc, option) => ({
    ...acc, [option.value]: rasters[option.value]
  // eslint-disable-next-line no-mixed-operators
  }), {})) || {};
```

Reply:
ok, will revert the change then

Reply:
@aabdaab done. Thanks for the catch